### ISSUE TITLE
jovo get fails with "TypeError: Cannot read property 'uri' of undefined"

### ISSUE THREAD

**Issue by serg06** [jovotech/jovo-cli#25]
```
> jovo get alexaSkill --skill-id <redacted>

? Found existing project files. How to proceed? Overwrite
   × Getting Alexa Skill project for ASK profile default
     -> Cannot read property 'uri' of undefined
     Getting Alexa Skill model files and saving to /platforms/alexaSkill/models

{ TypeError: Cannot read property 'uri' of undefined
    at Object.getSkillSimpleInformation (C:\Users\redacted\AppData\Roaming\npm\node_modules\jovo-cli\helper\alexaUtil.js:413:65)
    at p.then.then.then.then.then.then (C:\Users\redacted\AppData\Roaming\npm\node_modules\jovo-cli\commands\tasks.js:103:57)
    at <anonymous>
    at process._tickDomainCallback (internal/process/next_tick.js:228:7)
  context:
   { skillId: '<redacted>',
     locales: [ 'en-US' ],
     type: 'alexaSkill',
     target: 'all',
     projectId: undefined,
     stage: undefined,
     askProfile: 'default',
     info: 'Info: ' } }
```

I just set up ASK CLI by following [the instructions](https://developer.amazon.com/docs/smapi/set-up-credentials-for-an-amazon-web-services-account.html) to a dot. Not sure why it's not working, jovo new/init/build work fine. 

`Jovo CLI Version: 1.1.1`

**Comment by jankoenig** [comment]
Thanks. Did you set an endpoint for the Skill already? The CLI seems to be expecting that an endpoint is already set.

**Comment by jankoenig** [comment]
Hi there, thanks for helping us discover a bug: For me, it throws an error when the endpoint is empty. Will look into this with @aswetlow.

However, it works for me when I use a Lambda ARN. Will get back to you when we have investigated the problem.

**Comment by aswetlow** [comment]
Can't reproduce it with a Lambda ARN. Fixed the issue (in v1.1.3) when the endpoint is empty. Could you try it again, @serg06 ?

**Comment by serg06** [comment]
Works in 1.1.3 for me (though I have changed a lot of settings by now), thanks!
